### PR TITLE
README Improvements

### PR DESCRIPTION
**Pull request by pooky-programs**
- Added Note and generally clearer language in Getting Started for the purpose of directing users immediately to subtitle use if they wish
- Added basic regex explanation, use instructions, and regex materials

 - Added tutorial video that covers installation, setup, and basic use of asbplayer when used as a Japanese vocabulary mining tool, including Yomitan and Anki.
 - Improved language in the note under Getting Started for better understanding of use-cases
 - Added year of publication to other community guide videos

### CODE REVIEW

File: README.md
Comment (killergerbah) [inline README.md]: I think this line would be use case for the note syntax described here: https://github.com/orgs/community/discussions/16925

Also, I understand what you mean by "deck mining" but strictly speaking it should be "sentence mining" or "flashcard creation."

File: README.md
Comment (killergerbah) [inline README.md]: It's true that the information about the subtitle features can be found under "Detailed usage" but the reader still has to read and hunt for it since a lot of the documentation is still about making flashcards. So I would suggest these changes:
- Be more specific, for example: "just follow step 6."
- Make the "subtitle features" below "Detailed usage" more obvious such as by grouping them together or adding more information. Currently the only features explicitly documented are subtitle detection, offset, and regex filtering.

File: README.md
Comment (killergerbah) [inline README.md]: The phrase "Filtering subtitles" is a bit ambiguous IMO, as it could mean "filtering individual subtitles" or "filtering subtitle text."

File: README.md
Comment (killergerbah) [inline README.md]: Same comment as above about "filtering subtitles."
Also I would not consider "regexing" to be a commonly-used expression, even for more technical people.

> asbplayer will match any sequence following regex pattern and remove the matches.
- "a regex pattern" or "a specified regex pattern"
- asbplayer does not necessarily "remove the matches" even though that is a common use-case. Strictly speaking asbplayer replaces matched text with specified replacement text, which you have detailed below. So I at the very least I would say "can match ... and remove" instead of "will match ... and remove."

File: README.md
Comment (killergerbah) [inline README.md]: Same comment as about "regexing." I think it's clearer/more helpful to say "writing regular expressions" or similar.

**Review by killergerbah — CHANGES_REQUESTED** [review]

File: README.md
Comment (pooksh) [reply README.md]: OK, got it, making these changes now.
Should I mention that there is a difference between English () and Japanese（）?
something like
> Note: Japanese parentheses are usually different characters than the ones found on english keyboards: "()" versus "（）"

Or should I leave that to the user to discover since this is already kind of a niche thing?

**Comment by pooky-programs** [comment]
I got feedback from one of my peers on this PR and they suggested either labeling one of the guide videos as latest or tagging each guide with a version, rather than a year to imply recency (because a year is a long time). I like the latter but it might be a bit of trouble finding out the version of asbplayer at the time each was published.
Here are their own words:
> It can be confusing to update the community guide video links to say a year, this may imply they were newly updated. If the intention is to emphasize the newer tutorial, maybe list it as “latest” or the versions of the project when the video was made. 

What do you think?

File: README.md
Comment (killergerbah) [inline README.md]: > (in which case you will need the subtitle functionality to mine vocabulary or sentences)

Nitpick: I don't think this blurb is necessary to understand the note.

File: README.md
Comment (killergerbah) [inline README.md]: Since the value in providing multiple guides is mostly in the varying content and presentation, I don't think it's particularly valuable to annotate a single video as being the "latest" one. For example, people studying European languages will ignore any video that's marked Japanese, regardless of whether it's marked "Latest" or not.

File: README.md
Comment (killergerbah) [inline README.md]: I don't think this necessarily makes sense directly under `Detailed usage` since there are plenty of users who don't install the extension to use asbplayer with local files.

File: README.md
Comment (killergerbah) [inline README.md]: This isn't actually accurate, since asbplayer can be used without installing the extension. This would be more accurate:

> In order to make use of any of asbplayer's features, subtitles (or an empty subtitle track) must first be loaded, either onto a streaming video or a local video file. The following section instructs how to do this.

Even though using more words might be more precise in meaning, in general I think it's clearer to use less, more simple words since it's easier on the reader's eyes. Many people reading this documentation are not English natives so we should try to make it easy for them to read as well.

File: README.md
Comment (killergerbah) [inline README.md]: For the sections under `Subtitle setup and features` I would use smaller headings to indicate that each subsection belongs to this larger section. So use `####` instead of `###`. Similar for other subsections.

To be consistent with the section below about local files I would say:

> ### Enhancing streaming video with asbplayer-controlled subtitles

File: README.md
Comment (killergerbah) [inline README.md]: Nitpick: "asbplayer-controlled" instead of "asbplayer controlled"

File: README.md
Comment (killergerbah) [inline README.md]: I think this note about parantheses can be removed. Many people reading this don't care about Japanese and I'd prefer asbplayer to be seen as language-agnostic.

File: README.md
Comment (killergerbah) [inline README.md]: Not totally accurate, the website will have that button only if subtitles have been loaded into it somehow.

**Review by killergerbah — CHANGES_REQUESTED** [review]

**Review by killergerbah — CHANGES_REQUESTED** [review]
Thanks for going back-and-forth with me on this. Since it's very important for the README to be as useful and clear as possible I am being very opinionated in this review. I hope you understand.

**Comment by pooky-programs** [comment]
It's no problem, I totally understand the value in accuracy, good flow, and clear language when it comes to this sort of thing.

File: README.md
Comment (killergerbah) [inline README.md]: We can make this shorter. e.g:
"asbplayer is both a subtitle control and flashcard creation tool. If are not interested in flashcards, and only want to use asbplayer's subtitle features, you can read about them under [Subtitle setup and features](#subtitle-setup-and-features)."

Side note: it's easier to understand what "them" means when "asbplayer's subtitle features" is closer to it.

File: README.md
Comment (killergerbah) [inline README.md]: This can be shorter:
"Use one of the community guides to learn how to set up an Anki deck."

We are already telling the user to setup an Anki deck, so I don't think we need to repeat that instruction.

File: README.md
Comment (killergerbah) [inline README.md]: Let's use the more common word "website" and not "webtool."
I think we can also explain when to use the extension or when to use the website e.g:
"To use asbplayer with streaming video, install the Chrome extension. Otherwise, use the website."

File: README.md
Comment (killergerbah) [inline README.md]: Suggestion to shorten and use "website" instead of "webtool": 
"Once loaded into the extension, you can download the subtitles by opening the side panel and clicking the `Download Subtitles as SRT` button in the top-right. You can also download subtitles via the website by clicking the same download button in the top-left."

I would also remove the link to the website. Linking the website here may be a bit misleading since subtitles can't be downloaded simply by just accessing the website.

File: README.md
Comment (killergerbah) [inline README.md]: Can be shorter:
> Note: The [regex feature](#filtering-subtitle-text) will alter the `.srt` that is downloaded.

File: README.md
Comment (killergerbah) [inline README.md]: Shorten:
"Useful regular expressions:"

And specify the regex you're referring to in each bullet point.

File: README.md
Comment (killergerbah) [inline README.md]: Shorten: "Subtitle features"

File: README.md
Comment (killergerbah) [inline README.md]: Let's limit the resources we provide to a single high-quality one that doesn't have ads. Most people who know nothing about regexes will likely not want to click through multiple links to just learn about regexes. I suggest this one: https://regexlearn.com/playground

**Review by killergerbah — CHANGES_REQUESTED** [review]

File: README.md
Comment (pooksh) [reply README.md]: this is a wonderful resource!!

**Review by killergerbah — CHANGES_REQUESTED** [review]
Thanks for making all of the changes I requested. One more final opinionated nitpick: can you remove the triangles next to each subheading? I think it's easier to read without them. After that I'll merge the PR. Thanks again.

**Comment by killergerbah** [comment]
Thank you!!

**Comment by pooky-programs** [comment]
Yay!! So happy it got merged, thank you!!

**Pull request merged**
